docs/docsite/rst/porting_guides/porting_guide_4.rst

```diff
@@ -613,7 +652,7 @@ Ansible-core
 ~~~~~~~~~~~~
 
 - Starting in 2.14, shell and command modules will no longer have the option to warn and suggest modules in lieu of commands. The ``warn`` parameter to these modules is now deprecated and defaults to ``False``. Similarly, the ``COMMAND_WARNINGS`` configuration option is also deprecated and defaults to ``False``. These will be removed and their presence will become an error in 2.14.
-- apt_key - the parameter ``key`` does not have any effect, has been deprecated and will be removed in ansible-core version 2.14 (https://github.com/ansible/ansible/pull/70319).
+- apt_key - the paramater ``key`` does not have any effect, has been deprecated and will be removed in ansible-core version 2.14 (https://github.com/ansible/ansible/pull/70319).
```

Comment:
Unfortunate typo here

Reply:
a couple of comments :

- original spelling is correct
- we can't fix spelling errors in this file. The come from the collection itself. Anything we fix here will just go away in the next autogeneration of the porting guide :-( We're kind of stuck with that.